mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -1441,4 +1460,33 @@ class GeneralizedLinearRegressionTrainingSummary private[regression] (
         "No p-value available for this GeneralizedLinearRegressionModel")
     }
   }
+
+  /**
+   * Summary table with feature name, coefficient, standard error,
+   * tValue and pValue.
+   */
+  @Since("2.2.0")
+  lazy val summaryTable: DataFrame = {
```

Comment:
Could we have a better name? What about output model summary like R? Then we can directly override ```toString``` function?